tests/experimental/datasources/test_config.py

```diff
@@ -166,10 +181,26 @@ def test_yaml_file_config_round_trip(
     assert from_yaml_gx_config == re_loaded
 
 
+# TDD Tests for future work
+
+
 @pytest.mark.xfail(reason="Key Ordering needs to be implemented")
 def test_yaml_config_round_trip_ordering(
     inject_engine_lookup_double, from_yaml_gx_config: GxConfig
 ):
     dumped: str = from_yaml_gx_config.yaml()
 
     assert PG_CONFIG_YAML_STR == dumped
+
+
+@pytest.mark.xfail(
+    reason="Custom BatchSorter serialization logic needs to be implemented"
+)
```

Comment:
Should we add any of the other named args?

Reply:
Just updated the test to be better and more explicit.

https://github.com/great-expectations/great_expectations/pull/6580/commits/c2770d5e3144c02c4e2dd2dcec58db57b34d93f8